maddie/project-euler/euler_nine.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+
+def pythag_triplet():
+    for a in range(1, 1000):
+        for b in range(a + 1, 1000 - a):
+            c = 1000 - a - b
+
+            if a**2 + b**2 == c**2:
+                print("a = %d, b = %d, c = %d. abc = %d" % (a, b, c, a * b * c))
```

Comment:
Don't do this. Return the actual a, b, c, then print it outside the function. Functions should be pure.

https://medium.com/better-programming/what-is-a-pure-function-3b4af9352f6f